### ISSUE TITLE
How to merge cells on certain column vertically?

### ISSUE THREAD

**Issue by faradoxuz** [dolanmiu/docx#252]
Is there any method for merging cells vertically with examples?
In my opinion, there should be method getColumn() too)

**Comment by dolanmiu** [comment]
Only horizontal merging currently

This is a planned feature though

**Comment by faradoxuz** [comment]
thnx, I will wait for it

**Comment by vsingh7** [comment]
> Only horizontal merging currently
> 
> This is a planned feature though

I really hope so! This is the last feature I need for the app I'm working on 🙏 

Thanks so much for such an awesome library :)

**Comment by dolanmiu** [comment]
Will be fixed with:

https://github.com/dolanmiu/docx/pull/274

**Comment by vsingh7** [comment]
I'm so sorry but I can't seem to get a reference to a column! It doesn't seem like any of the files that were committed in #274 that pertain to `table-column` are showing up when I `npm install docx@4.7.1`. This ultimately leads me to the error of `TypeError: table.getColumn is not a function`.

Any insight would be greatly appreciated!

**Comment by dolanmiu** [comment]
#274 is in docx@5.0.0-rc1, so try installing with:

```
$ npm install docx@next
```

https://github.com/dolanmiu/docx/releases/tag/5.0.0-rc1

**Comment by vsingh7** [comment]
Thank you for the fast reply!

I tried that and I also tried `npm install docx@5.0.0-rc1` but still no go. Every time I try to update and then follow with `npm show docx version` it always says `4.7.1`. Even though the correct version isn't being returned, I can now see the relevant `table-column` files in my `node_modules`, so something somewhere changed. I'm just not sure what!

Maybe it is worth mentioning that I'm trying to implement this on the front-end and not the back-end? Not sure if that makes a difference at all. Here's the code I'm trying (adapted from one of your examples):

```js
const doc = new docx.Document();

const table = doc.createTable({
    rows: 4,
    columns: 4,
});
table.getCell(2, 2).addParagraph(new Paragraph("Hello"));
table.getColumn(3).mergeCells(1, 2);

let packer = new docx.Packer(doc);
let filename = "Table of Objectives - " + this.title + ".docx";

packer.toBlob(doc).then(blob => {
    save(blob, filename);
});
```

And from that I get this error:
```
Error: Index out of bounds when trying to get row on table
```

I know I'm doing something wrong but I'm not sure what!
